setup.py

```diff
@@ -102,6 +103,7 @@ def build_extensions(self):
         "biopython>=1.73",  # pyfaidx needs this for reading bgzipped FASTA files
         "scipy",
         "xopen>=1.2.0",
+        "pulp>=2",
```

Comment:
Yes. Even for whole chromosomes the running time was absolutely fine. If the solver was not sufficient, I would not use an ILP in the first place, because cross-linking another (proprietary) solver is quite a pain and would make the tool too difficult to access for users. Should PuLP ever break, it is possible to solve the underlying problem with some flow-based graph algorithm in polynomial time, but it was much easier to set up a small ILP than diving into some flow implementations.

Reply:
Great, nice to not have to rely on proprietary solvers!